README.md

```diff
@@ -1,7 +1,14 @@
-activate venv  
-Install requirements - `pip install -r requirements.txt`  
-Start web server - `uvicorn src.main:app --reload`  
+# pyndemic web backend
+## fastapi app
+* Activate/create venv 
+* Install requirements - `pip install -r requirements.txt`
+* Add `Pyndemic` engine to `PYTHONPATH`
```

Comment:
Pyndemic dependency already exists in requirements .txt

Reply:
Ok, I have fixed it